joss/paper.md

```diff
@@ -0,0 +1,103 @@
+tags:
+  - law
+  - courts
+  - Python
+authors:
+  - name: Jack Cushman
+    affiliation: 1
+  - name: Matthew Dahl
+    affiliation: 2
+  - name: Mike Lissner
+    affiliation: 3
+affiliations:
+ - name: Harvard University, Berkman Klein Center
+   index: 1
+ - name: University of Notre Dame, Department of Political Science
+   index: 2
+ - name: Free Law Project
+   index: 3
+date: 7 July 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`eyecite` is a Python package for parsing and extracting legal citations from
+text. It can recognize a wide variety of citations commonly appearing in American
+legal decisions, including:
+
+- full case (e.g., `Bush v. Gore, 531 U.S. 98, 99-100 (2000)`)
+- short case (e.g., `531 U.S., at 99`)
+- statutory (e.g., `Mass. Gen. Laws ch. 1, § 2`)
+- law journal (e.g., `1 Minn. L. Rev. 1`)
+- supra (e.g., `Bush, supra, at 100`)
+- id (e.g., `Id., at 101`)
+
+It also offers tools for efficiently aggregating like citations and annotating
+citations with custom markup.
+
+# Statement of need
+
+Citations are the bedrock of legal writing and a frequent topic of legal
+research, but few open-source tools exist for extracting them from legal
+texts. Because of this, researchers have historically relied on proprietary
+citation data provided by vendors like LexisNexis and Westlaw [e.g., @Black2013;
+@Fowler2007; @SpriggsII2000] or have used their own personal scripts to parse
+such data from texts ad hoc [e.g., @Clark2012; @Fowler2008]. By providing an
+open-source, standardized alternative to these approaches, `eyecite` promises
+to increase scholarly transparency and consistency. It also promises to give
+researchers the extendability and flexibility to develop new methods of
+citation analysis that are currently not possible under the prevailing
+approaches.
+
+For example, one burgeoning research agenda seeks to apply machine learning
+techniques to citation analysis, either to recommend relevant authorities to
+legal practitioners [@HoForthcoming], model the topography of the legal search
+space [@Dadgostari2021; @Leibon2018], or automatically detect and label
+the semantic purpose of citations in texts [@Sadeghian2018]. One obvious
+application of `eyecite` would be to use it to generate empirical training
+data for these kinds of machine learning tasks.
+
+To facilitate those kinds of projects and more, `eyecite` exposes significant
+entity metadata to the user. For case citations, `eyecite` parses and
+exposes information regarding a citation's textual position, year, normalized
+reporter, normalized court, volume, page, pincite page, and accompanying
+parenthetical text, as well as `eyecite`'s best guess at the names of the
+plaintiff and defendant of the cited case.  For statutory citations, `eyecite`
+parses and exposes information regarding a citation's textual position, year,
+normalized reporter, chapter, section, publisher, and accompanying parenthetical
+text. It does this using a
```

Comment:
Would a couple of pictures be useful here? Seems like a picture of a full citation with the different parts labeled would be really useful to prevent people's eyes from glazing over.

Reply:
I took a stab at creating a such a figure in the new draft, let me know what you think